lib/rules/common/require-type.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
```

Comment:
It might make more sense for these to be in the same directory as the rule they pertain to. This makes it look like there is a rule called `common`.

maybe make the rules a directory instead of single modules. `lib/rules/tap-consistent-assertions/index.js`

Reply:
hmm yeah I agree, I think I can just move `common` up a level? the `tap-synonyms` is specific to one rule, but `require-type` is shared between the other two.